fictadvisor-api/src/v2/api/controllers/ScheduleController.ts

```diff
@@ -300,6 +299,11 @@ export class ScheduleController {
     name: 'week',
     required: false,
   })
+  @ApiEndpoint({
+    summary: 'Get events by group',
+    guards: TelegramGuard,
+  })
```

Comment:
the same problem, this endpoint must be accessible from bot and also from web, so return permission